lib/stdlib/doc/src/orddict.xml

```diff
@@ -69,6 +71,22 @@
           generated if the initial value associated with <c><anno>Key</anno></c>
           is not a list of values.</p>
         <p>See also section <seeerl marker="#notes">Notes</seeerl>.</p>
+          <p><em>Example 1:</em></p>
+          <pre>
+1> <input>OrdDict1 = orddict:from_list([{a, 1}, {b, 2}]).</input>
+[{a,1},{b,2}]
+2> <input>OrdDict2 = orddict:append(c, 3, OrdDict1).</input>
+[{a,1},{b,2},{c,[3]}]
+3> <input>orddict:fetch(c, OrdDict2).</input>
+[3]
+</pre>
+          <p><em>Example 2:</em></p>
+          <pre>
+1> <input>OrdDict1 = orddict:from_list([{a, 1}, {b, 2}]).</input>
```

Comment:
It seems unnecessary to have two examples, unless you have some explaining text between the examples, which you don't have here. In this case, I think it would be easier to read if you combined the examples. That applies to many of the other examples.

Reply:
In this particular case, having two examples can be clearer, because you can start out with different initial orddicts to make it clearer what is happening.

Here is a suggestion:

**Example 1:**

```
1> OrdDict1 = orddict:from_list([{x, []}]).
[{x,[]}]
2> OrdDict2 = orddict:append(x, 1, OrdDict1).  
[{x,[1]}]
3> OrdDict3 = orddict:append(x, 2, OrdDict2).
[{x,[1,2]}]
4> orddict:append(y, 3, OrdDict3). 
[{x,[1,2]},{y,[3]}]
```

(Your example didn't really show any real appending being done, so I fixed that. By only having one key in the orddict, it is clear enough what is happening without having to use `orddict:fetch/2`.)

**Example 2:**
```
1> OrdDict1 = orddict:from_list([{a, no_list}]).
[{a,no_list}]
2> orddict:append(a, 1, OrdDict1).              
** exception error: bad argument
     in operator  ++/2
        called as no_list ++ [1]
```